packages/web-react/src/components/Checkbox/demo/Checkbox.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { StoryFn } from '@storybook/react';
+import Checkbox from '../Checkbox';
+import { SpiritCheckboxProps } from '../../../types';
```

Comment:
I would like to see imports sorted, but we can fix it when we make all those demos in the near future.

Reply:
sorted